example/spec/parallel/ingest/ingestFromPdrSpec.js

```diff
@@ -234,16 +230,8 @@ describe('Ingesting from PDR', () => {
         });
       });
 
```

Comment:
Yeah, it does get added but there is a race condition that I couldn't figure out how to resolve. Basically, because of the time when the PDR workflow is queued and executed is indeterminate, by the time we go to fetch the PDR model status that execution may be running or completed. So testing for the PDR record being "running" may intermittently fail because it is actually "completed". 

I compensated by adding the same test to the other ingest spec, even though that one isn't specifically for PDRs, because it doesn't use queueing and is more predictable:

https://github.com/nasa/cumulus/pull/1202/files#diff-45ed34e4b37c728d5f2e32f579c4cfc9R218

Reply:
Got it. Yeah I did see that. Ok just checking.